<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Classification Statistics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QgsMessageBar" name="mMessageBar" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mask</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QgsMapLayerComboBox" name="mMask">
         <property name="allowEmptyLayer">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QToolButton" name="mRun">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/themes/default/propertyicons/action.svg</normaloff>:/images/themes/default/propertyicons/action.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="mProgressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QgsMapLayerComboBox" name="mRaster">
           <property name="allowEmptyLayer">
            <bool>true</bool>
           </property>
           <property name="excludedProviders">
            <stringlist>
             <string>ogr</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsRasterBandComboBox" name="mBand"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Raster</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="mPlot">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsRasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsrasterbandcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>enmapboxapplications.imagestatistics.core</header>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QWidget</extends>
   <header>qgis._gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mRaster</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>mBand</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
